community-members.md

```diff
@@ -91,6 +91,7 @@ Repo: [open-telemetry/opentelemetry-js](https://github.com/open-telemetry/opente
 Approvers:
 
 - [Yuri Shkuro](https://github.com/yurishkuro), Uber
+- [Isobel Redelmeier](https://github.com/iredelmeier), LightStep
```

Comment:
(please remove these other files from the PR)

Reply:
merge incident. Fixed